spdx-utils/src/main/antlr/org/ossreviewtoolkit/spdx/SpdxExpression.g4

```diff
@@ -56,7 +51,7 @@ simpleExpression
 compoundExpression
     :
     simpleExpression
-    | simpleExpression WITH licenseExceptionExpression
+    | simpleExpression WITH (IDSTRING | LICENSEREFERENCE)
```

Comment:
Commit message: "all instances **of** `SpdxExpression` are syntactically correct."

Reply:
Done.